index.js

```diff
@@ -1,88 +1,121 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1;
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter++;
+
+console.log(counter); 
+ // => 2
+ console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+const name = "Seth";
+console.log(name);
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
-
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red")
+
+const drinkWater = currentThirstLevel => {
```

Comment:
Nice use of arrow functions! Totally not required now but good to get comfortable with them sooner than later.